coalib/output/printers/LogPrinter.py

```diff
@@ -115,7 +115,7 @@ def log_exception(self,
 
         self.log(log_level, message, timestamp=timestamp, **kwargs)
```

Comment:
it appeared the me that there was an empty line between the comma and the "it's", but now it doesnt. might have been from my side, ignore this.